plugins/net_plugin/net_plugin.cpp

```diff
@@ -1511,6 +1549,39 @@ namespace eosio {
       sync_wait();
    }
 
+   //-----------------------------------------------------------
+   void block_status_monitor::reset() {
+      in_accepted_state_ = true;
+      events_ = 0;
+   }
+
+   void block_status_monitor::accepted() {
+      if(in_accepted_state_) {
+         return;
```

Comment:
Not that performance is a real issue here, but this is likely to be faster to just set the values. Although the compiler has probably removed this if anyway. I think it is clearer to just set the values. But not a huge deal either way.

Reply:
I appreciate your comment and thinking.  I debated internally the difference between just resetting the values and testing the current state. I'll change it to just setting the values. 
My internal debate centered around wondering what the most likely state on each call.  If the most likely state is accepted then the if test would reduce cpu cycles with a hiccup on when the state changed.  If the state is continually going back and forth, then removing the if test would most likely reduce cpu cycles.